spec/std/isa/inst/Zicfiss/ssrdp.yaml

```
@@ -24,3 +24,20 @@ access:
   vu: always
 data_independent_timing: false
 operation(): |
+  Csr csr_handle = direct_csr_lookup(12'h011);
```

We could probably use a better way of looking up arbitrary CSRs, by name perhaps... ignoring that, could we maybe add a bit of context so it reads better?
```suggestion
  Bits<12> ssp_addr = 0x011;
  Csr csr_handle = direct_csr_lookup(ssp_addr);
```